overviewer_core/world.py

```diff
@@ -1710,7 +1710,8 @@ def get_chunk(self, x, z):
         if not 'sections' in chunk_data:
             # This world was generated pre 21w43a and thus most chunk data is contained
             # in the "Level" key
-            chunk_data = chunk_data['Level']
+            if 'Level' in chunk_data:
+            	chunk_data = chunk_data['Level']
```

Comment:
This seems to miss a few spaces? :thinking: 
```suggestion
            	   chunk_data = chunk_data['Level']
```